samples/GalleryApp/GalleryApp.csproj

```diff
@@ -115,9 +115,22 @@
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="..\Common\cs\FrameReaderFrameSource.cs">
+      <Link>FrameSource\FrameReaderFrameSource.cs</Link>
+    </Compile>
+    <Compile Include="..\Common\cs\IFrameSource.cs">
+      <Link>FrameSource\IFrameSource.cs</Link>
+    </Compile>
+    <Compile Include="..\Common\cs\ImageFileFrameSource.cs">
+      <Link>FrameSource\ImageFileFrameSource.cs</Link>
+    </Compile>
+    <Compile Include="..\Common\cs\MediaPlayerFrameSource.cs">
+      <Link>FrameSource\MediaPlayerFrameSource.cs</Link>
+    </Compile>
     <Compile Include="App.xaml.cs">
       <DependentUpon>App.xaml</DependentUpon>
     </Compile>
+    <Compile Include="BodyRenderer.cs" />
```

Comment:
>BodyRenderer.cs [](start = 22, length = 15)

can this be instead a reference to the existing BodyRender.cs?

Reply:
yes

---
In reply to: [297270849](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/35#discussion_r297270849) [](ancestors = 297270849,296978554)